tools/amd_build/pyHIPIFY/hipify_python.py

```diff
@@ -938,7 +962,8 @@ def get_kernel_template_params(output_directory, the_file, KernelDictionary, tem
             for idx, arg_type in enumerate(argument_types):
                 formatted_args[idx] = arg_type
 
-            KernelDictionary[kernel_name] = {"kernel_with_template": kernel_with_template, "arg_types": formatted_args}
+            # See Note [PyTorch and Caffe2 kernel name clobber]
```

Comment:
lol, good news is we will be able to get rid of this static_cast insertion step soon